README.md

```diff
@@ -9,15 +9,23 @@
 
 ---
 
-Our templates offer a rich boilerplate to jump start React-based application development for [Create React App](https://github.com/facebook/create-react-app).
+<p align="center">
+    <a href="https://www.npmjs.com/package/cra-template-nimble"><img src="https://badgen.net/npm/v/cra-template-nimble" /></a>
+    <a href="https://www.npmjs.com/package/cra-template-nimble"><img src="https://badgen.net/npm/dy/cra-template-nimble" /></a>
+</p>
+
+Our template offers a rich boilerplate to jump-start React-based application development for [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
wondering if it should be `with [Create React App]` or `from [Create React App]` instead of `for` 🤔  I think we are extending what the create-react-app offers, so I thought using "for" might be slightly odd 😛

Reply:
Definitely not `for` ^^
Both with and from seems good, but I would prefer "with" (it sounds like cra stands next to the project, not just at the origin — which is the case as cra is not ejected and remains a project dependency).,

Reply:
✅ 0ac66df